src/ameba/severity.cr

```diff
@@ -9,8 +9,12 @@ module Ameba
     # ```
     # Severity::Warning.symbol # => 'W'
     # ```
-    def symbol
-      to_s[0]
+    def symbol : Char
+      case self
+      in Error      then 'E'
+      in Warning    then 'W'
+      in Convention then 'C'
+      end
```

Comment:
this would have to raise in else branch.

Reply:
That's an exhaustive case (notice `in` instead of `when`) so there's no option for an `else` branch.